src/server/.tests/routes/controllers/Middleware/isUserAuthenticated.spec.js

```diff
@@ -0,0 +1,53 @@
+const isUserAuthenticated = require('../../../../routes/controllers/Middleware/isUserAuthenticated')
+const assert = require('chai').assert
+const sinon = require('sinon')
+const httpMocks = require('node-mocks-http')
+const EventEmitter = require('events').EventEmitter
+
+let suite = {}
+
+describe('isUserAuthenticated', () => {
+	beforeEach(() => {
+		suite = {}
+		suite.isUserAuthenticatedMock = isUserAuthenticated(suite.passportMock)
+		suite.nextSpy = sinon.spy()
+		suite.responseMock = httpMocks.createResponse({
+			eventEmitter: EventEmitter
+		})
+	})
+	it('should skip to next middleware by calling next() when user is already logged in ', () => {
+		//given
+		const requestMock = {
+			isAuthenticated: () => true
+		}
+
+		//when
+		suite.isUserAuthenticatedMock(requestMock, suite.responseMock, suite.nextSpy)
```

Comment:
The purpose of test is never to test mock. It's not mock BTW.

Reply:
Fix in other tests

Reply:
ok